### PR TITLE
Fixed some typos.

### PR DESCRIPTION
**Pull request by haosdent**

### CODE REVIEW

File: tensorflow/cc/framework/gradient_checker.cc [hunk 1]
```diff
@@ -47,7 +47,7 @@ Status ComputeTheoreticalJacobianTranspose(
   auto dy_data_flat = dy_data.flat<T>();
   dy_data_flat.setZero();
 
-  // Compute the theoretical Jacobian one row at a time by backproping '1.0'
+  // Compute the theoretical Jacobian one row at a time by back probing '1.0'
```
Comment (asimshankar) [inline tensorflow/cc/framework/gradient_checker.cc]: I believe this is meant to be "backproping" as in "back propagating" and not "back probing"

**Review by asimshankar — APPROVED** [review]

**Comment by asimshankar** [comment]
Mr Jenkins, test this please.

**Comment by asimshankar** [comment]
Thanks for the fixes! Should be merged once tests pass.

**Comment by haosdent** [comment]
Many thanks!

**Pull request merged**